web-service/helpers.py

```diff
@@ -189,12 +196,8 @@ def GetContentEncoding(content):
             encoding = value[0]
```

Comment:
For info, the encoding is not passed as parameter of  `GetContentEncoding`. 
What is passed is the page content which is used to detect the encoding of it.